### ISSUE TITLE
get_animals don't filter on animal_project

### ISSUE THREAD

**Issue by damianooldoni** [inbo/etn#75]
Bug found by @peterdesmet. Thanks for it. Indeed, I confirmed that `get_animals` returns too much animals as it doesn't filter properly.

Example:
```r
animals <- get_animals(connection = con, animal_project = "2011_rivierprik")
```
`animals` should contain 39 lines, but it is not.

The problem is that we have param `netework_project` in the query too and if no network is specified than we take all network projects. This should be avoided. Removing `network_project` param from the function and the code related to it seems to solve the issue.
I am immediately tackling this bug in branch [no-ntwk_prjct-in-get_animals](https://github.com/inbo/etn/tree/no-ntwk_prjct-in-get_animals). Asap to master.

**Comment by damianooldoni** [comment]
This issue seems vaguely related to #43.

**Comment by damianooldoni** [comment]
Solved by #76.
